src/reducers/entities/posts.js

```diff
@@ -411,7 +443,9 @@ export default function(state = {}, action) {
         // Object mapping post ids to post objects
         posts,
 
-        // Object mapping channel ids to an list of posts ids in that channel with the most recent post first
+        pendingPostIds: handlePendingPosts(state.pendingPostIds, action),
```

Comment:
Maybe just add a quick comment saying what this is like the other reducers

Reply:
@jwilander done